db/migrate/20230616100340_add_final_publish_to_decidim_participatory_documents_documents.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddFinalPublishToDecidimParticipatoryDocumentsDocuments < ActiveRecord::Migration[6.0]
+  def change
+    add_column :decidim_participatory_documents_documents, :final_publish, :boolean, default: false
```

Comment:
```suggestion
    add_column :decidim_participatory_documents_documents, :published_at, :datetime, index: true
```
I prefer that we use a more standard field here. Also, by using a date we can get information about the time it has been published that might com in handy in the future.